osu.Game/Overlays/Profile/Sections/Ranks/DrawableProfileScore.cs

```diff
@@ -142,17 +143,21 @@ private void load()
                         {
                             new Box
                             {
+                                Anchor = Anchor.TopRight,
+                                Origin = Anchor.TopRight,
                                 RelativeSizeAxes = Axes.Both,
-                                Size = new Vector2(1, 0.5f),
+                                Size = new Vector2(1.06f, 0.5f),
```

Comment:
How was this size determined? Maybe it should be absolute?

Reply:
Nice catch. I agree it's better to make this absolute, because changing `performance_width` and `DrawableProfileScore.Height` shows that using relative widths isn't very robust. Currently working on a solution.